overlord/state/locktrace_fake.go

```diff
@@ -0,0 +1,28 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+//go:build !statelocktrace
+
+/*
+ * Copyright (C) 2021 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package state
+
+func lockTimestamp() int64 {
+	return int64(0)
+}
+
+func maybeSaveLockTime(lockWaitStart, lockHoldStart, now int64) {
+}
```

Comment:
tiny nitpick, we can simplify the no-ops
```suggestion
func lockTimestamp() int64 { return 0 }
func maybeSaveLockTime(int64, int64, int64) {}
```